test/publish.bats

```diff
@@ -243,3 +243,28 @@ published:
 EOF
 
 }
+
+@test "building from published images with whiteouts" {
```

Comment:
Could we describe the test and what we expect and the observed failure here in comments?